demo/src/app/components/shared/api-docs/api-docs.component.html

```diff
@@ -1,12 +1,15 @@
 <div class="api-doc-component">
   <h2>
+    {{apiDocs.className}}
     <a href="https://github.com/ng-bootstrap/ng-bootstrap/tree/master/{{apiDocs.fileName}}"
        target="_blank"
        angulartics2On="click"
        angularticsEvent="Demo HTML Content"
-       angularticsCategory="{{ demoTitle }}">{{apiDocs.className}}</a>
+       angularticsCategory="{{ demoTitle }}">
```

Comment:
I don't know that I love the text link here.  Maybe an icon?  A GH icon?  I'm not married to the component title being a hyperlink, but do agree that we should link to the source on GH....just not sure how.